pkg/update/required.go

```diff
@@ -119,3 +151,16 @@ func getNonDeployableCause(requiredUpdates []string) string {
 	}
 	return fmt.Sprintf("This version cannot be deployed because versions %s are required and must be deployed first.", versionLabelsStr)
 }
+
+// getKubeVersionNonDeployableCause constructs a non-deployable cause message based on the kube range validation error message
+func getKubeVersionNonDeployableCause(err error) string {
+	switch {
+	case errors.Is(err, util.ErrKubeMinorRangeMismatch):
+		return "Before you can update to this version, you need to update to an earlier version that includes the required infrastructure update."
+	case errors.Is(err, util.ErrKubeVersionDowngrade):
+		return "Release includes a downgrade of the infrastructure version, which is not allowed. Cannot use release."
+	case errors.Is(err, util.ErrKubeMajorVersionUpgrade):
+		return "Release includes a major version upgrade of the infrastructure version, which is not allowed. Cannot use release."
```

Comment:
I wonder if we even need this given that another major version of kubernetes doesn't exist. like how could this even happen right now?